lib/api/v3/work_packages/form/work_package_payload_representer.rb

```diff
@@ -77,7 +77,8 @@ def initialize(represented, options = {})
           property :done_ratio,
                    as: :percentageDone,
                    getter: -> (*) { done_ratio if Setting.work_package_done_ratio != 'disabled' },
```

Comment:
@myabc You are right, now that it doesn't show anymore when disabled I can probably remove the whole getter part.